mindmap/mindmap.py

```diff
@@ -2,27 +2,41 @@
 
 from __future__ import annotations
 
```

Comment:
Here are the substantial changes! If you get lost in all the changes, most are miscellaneous.